app/src/main/java/net/gsantner/opoc/util/GsFileUtils.java

```diff
@@ -61,8 +66,8 @@ public class GsFileUtils {
     public final static SimpleDateFormat DATEFORMAT_IMG = new SimpleDateFormat("yyyyMMdd-HHmmss", INITIAL_LOCALE); //20190511-230845
 
     // Used on methods like copyFile(src, dst)
-    private static final int BUFFER_SIZE = 4096;
-    private static final GsHashMap<String, String> MIME_TYPE_CACHE = new GsHashMap<>();
```

Comment:
This is just a path -> mime cache. IMO many thousands of entries will not affect performance. I think we can just use a ConcurrentHashMap and we will gain some speed and safety